pkg/minikube/constants/constants.go

```diff
@@ -60,14 +60,14 @@ func MakeMiniPath(fileName ...string) string {
 	return filepath.Join(args...)
 }
 
-// GetProfileHomeDir determines the Minishift home directory for the profile.
-func GetProfileHomeDir() string {
-	if ProfileName != DefaultProfileName {
+// GetProfileHomeDir determines the base directory for the specified profile name
+func GetProfileHomeDir(profile string) string {
```

Comment:
👍

Reply:
using constants in methods is bad. We should rather pass parameters. The more functions rely internally on constants, the tighter the coupling gets and the harder to change.